ui/src/providers/experiment/context.js

```diff
@@ -23,38 +19,24 @@ export const ExperimentContextProvider = ({ projectId, children }) => {
     [appConfig]
   );
 
-  const [{ data: { data: experiments, paging }, isLoaded }] = useXpApi(
+  const [{ data: { data: experiments }, isLoaded }] = useXpApi(
     `/projects/${projectId}/experiments`,
     {
       query: {
         start_time: start_time,
         end_time: end_time,
-        page: pageIndex + 1,
-        page_size: appConfig.pagination.defaultPageSize,
-        status: "active"
+        fields: appConfig.listExperimentFields.experimentContextFields,
+        status_friendly: ["running", "scheduled"]
       },
     },
-    { data: [], paging: { total: 0 } }
+    { data: [] }
   );
 
-  useEffect(() => {
-    if (isLoaded) {
-      if (!!experiments && !isAllExperimentsLoaded) {
-        setAllExperiments((curExperiments) => [...curExperiments, ...experiments]);
-      }
-      if (paging.pages > paging.page) {
-        setPageIndex(paging.page);
-      } else {
-        setIsAllExperimentsLoaded(true);
-      }
-    }
-  }, [isLoaded, experiments, paging, isAllExperimentsLoaded]);
-
   return (
     <ExperimentContext.Provider
       value={{
-        allExperiments,
-        isAllExperimentsLoaded,
+        experiments,
```

Comment:
Should we rename this to `scheduledAndRunningExperiments` or something to be more specific? Although the `ExperimentContextProvider` is only used in the Turing microfrontend components now, a more accurate name could help development in the future if we look to use it in other places.